argus/scenarios/cloud/windows.py

```diff
@@ -14,9 +14,30 @@
 #    under the License.
 
 import collections
+from six.moves.urllib.parse import urlparse

```

Comment:
`argus/scenarios/cloud/windows.py:17:1: H302  import only modules.'from six.moves.urllib.parse import urlparse' does not import a module` this is what I got.